src/controllers/handlers/delete-community-handler.ts

```diff
@@ -0,0 +1,57 @@
+import { HandlerContextWithPath, HTTPResponse } from '../../types'
+import { messageErrorOrUnknown } from '../../utils/errors'
+import { CommunityNotFoundError } from '../../adapters/errors'
+import { NotAuthorizedError } from '@dcl/platform-server-commons'
+import { isOwner } from '../../logic/community'
+
+export async function deleteCommunityHandler(
+  context: Pick<
+    HandlerContextWithPath<'communitiesDb' | 'logs', '/communities/:id'>,
+    'components' | 'params' | 'verification'
+  >
+): Promise<HTTPResponse> {
+  const {
+    components: { communitiesDb, logs },
+    params: { id },
+    verification
+  } = context
+  const logger = logs.getLogger('privacy-handler')
+  logger.info(`Deleting community: ${id}`)
+
+  const userAddress = verification?.auth.toLowerCase()
```

Comment:
if singed fetch is required on this endpoint we will always have a verification.auth so we can do:

`verification!.auth` and then only validate the address with `EthAddress.validate`